apps/laboratory/next-env.d.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-import './out/types/routes.d.ts'
+import "./out/types/routes.d.ts";
```

Comment:
This started happening to me as well - prettier not happy with it, we should prob ignore this on prettierignore